src/styles/blocks/image.scss

```diff
@@ -0,0 +1,10 @@
+.image {
+  padding: 32px 19px 40px;
```

Comment:
Consider defining the padding values as variables in a separate file to adhere to the task requirements for maintainability.